knapsack_01.ipynb

```diff
@@ -0,0 +1,90 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "fe5e192a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Time complexity: O(n^2)\n",
```

Comment:
There is no memoization happening here. So it becomes exponential. Try memoization.